src/cmd/singularity/cli/keys_pull.go

```diff
@@ -18,7 +18,7 @@ import (
 
 func init() {
 	KeysPullCmd.Flags().SetInterspersed(false)
-	KeysPullCmd.Flags().StringVarP(&url, "url", "u", "", "overwrite the default remote url")
+	KeysPullCmd.Flags().StringVarP(&keyServerURL, "url", "u", "", "specify the key server URL (default: https://keys.sylabs.io)")
```

Comment:
Should `pull` also read key server default from config file like `sign` and `verify`?

Reply:
This is removed completely see my comment above.